Dockerfile.armhf

```diff
@@ -15,7 +15,7 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM armhf/debian:jessie
+FROM armhf/debian:stretch
```

Comment:
armhf was also deprecated (see https://hub.docker.com/r/armhf/debian/) but I only see an arm32v5 and a arm32v7 debian image. 

@tianon is there a reason there is no arm32v6 debian image, I think that is probably what should be here? (I triple checked it doesn't exist this time 😄 )